legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/AbstractLSPGrammarExtension.java

```diff
@@ -253,6 +253,11 @@ protected void collectCommands(SectionState sectionState, PackageableElement ele
         this.commandsSupports.forEach(x -> x.collectCommands(sectionState, element, consumer));
     }
 
+    public Iterable<? extends LegendExecutionResult> defaultExecute(SectionState section, String entityPath, String commandId, Map<String, String> executableArgs, Map<String, Object> inputParameters)
```

Comment:
Dont think we should create this method.

Reply:
Removed it